main.go

```diff
@@ -1,6 +1,9 @@
 package main
 
-import "github.com/bitrise-core/bitrise-plugins-analytics/cli"
+import (
+	"github.com/bitrise-core/bitrise-plugins-analytics/cli"
+	_ "github.com/stretchr/testify/require"
```

Comment:
Require package is now imported to the root scope. I think this is wrong this way.

Reply:
How should it be imported? The `_tests/integration` package is not realized by dep.

Reply:
I think simply including in the `Gopkg.toml` should be enough. So when updating the deps it will be downloaded as well. Only `dep init` will not include packages from test files, however you can add any you want.

Reply:
That case we would need to remove 

```
[prune]
  unused-packages = true
```

Reply:
I see. I still have major concerns about importing a package with _ to root level. This way the package is totally unused and if there is init() in he package then that will be called for sure.